docs/source/notebooks/subset.ipynb

```diff
@@ -109,15 +128,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 6,
    "id": "961b3e52",
    "metadata": {
     "execution": {
-     "iopub.execute_input": "2022-01-24T17:53:50.237251Z",
-     "iopub.status.busy": "2022-01-24T17:53:50.236525Z",
-     "iopub.status.idle": "2022-01-24T17:53:50.956633Z",
-     "shell.execute_reply": "2022-01-24T17:53:50.955914Z"
-    }
+     "iopub.execute_input": "2022-02-04T18:11:35.479101Z",
+     "iopub.status.busy": "2022-02-04T18:11:35.477269Z",
+     "iopub.status.idle": "2022-02-04T18:11:38.681044Z",
+     "shell.execute_reply": "2022-02-04T18:11:38.682228Z"
+    },
+    "tags": [
+     "nbval-ignore-output"
```

Comment:
whooo, fancy !, I'll test this in a Jenkins run to be sure.

When we simply open the notebook, do we see this tag?  Or it is hidden so unless we search, we do not know which cells have this tag and which do not.  Could potentially be a hidden debugging problem later.  But yeah we can try.

Reply:
On the jupyter notebook interface: View -> Cell toolbar -> Edit Metadata. And then on each cell a "edit" button appear. When clicked it shows the JSON part with that tag.
I agree it isn't evident and might make debugging harder.